installer/sharch_body.sh

```diff
@@ -27,6 +27,7 @@ echo " OK."
 
 # Untar and launch install script in a tmpfs
 cur_wd=$(pwd)
+export cur_wd
```

Comment:
A correction : the cur_wd is actually /sonic (from the Makefile's  -w /sonic ) and not the absolute build path..  Nevertheless, this path needs to be passed to the installer so that build_image.sh may directly retrieve it in-place after the "build" mode installation.